core/jobs/batch_jobs/suggestion_stats_computation_jobs_test.py

```diff
@@ -1225,9 +1248,11 @@ def test_creates_stats_model_from_multiple_suggestions(self) -> None:
         self.assertEqual(translation_stats_model.rejected_translations_count, 0)
         self.assertEqual(
             translation_stats_model.rejected_translation_word_count, 0)
-        self.assertItemsEqual(
+        self.assertListEqual(
```

Comment:
Can you add a comment here noting that this should be in ascending order?

Reply:
Done